### PR TITLE
WRQ-17541: Changed `ui/Marquee.MarqueeDecorator` to use `@enact/i18n` utils for `marqueeDirection`

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updates ui/Marquee.MarqueeDecorator.marqueeDirection to use enact/i18n utils.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updates ui/Marquee.MarqueeDecorator.marqueeDirection to use enact/i18n utils.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-17541

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Comment by MikyungKim** [comment]
@hong6316 , Could you resolve the merge conflicts, please?

File: packages/ui/Marquee/MarqueeDecorator.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* global ResizeObserver */
 
-import direction from 'direction';
+import {isRtlText} from '@enact/i18n/util';
```
Comment (MikyungKim) [inline packages/ui/Marquee/MarqueeDecorator.js:3]: This import statement is not been sorted alphabetically.

File: packages/ui/Marquee/MarqueeDecorator.js [hunk 2]
```diff
@@ -125,7 +125,7 @@ const defaultConfig = {
 	 * @kind member
 	 * @memberof ui/Marquee.MarqueeDecorator.defaultConfig
 	 */
-	marqueeDirection: (str) => direction(str) === 'rtl' ? 'rtl' : 'ltr'
+	marqueeDirection: (str) => isRtlText(str) ? 'rtl' : 'ltr'
```
Comment (MikyungKim) [inline packages/ui/Marquee/MarqueeDecorator.js:128]: If we change this, is it necessary to keep https://github.com/enactjs/sandstone/blob/master/Marquee/Marquee.js#L25 ?

File: packages/ui/package.json [hunk 3]
```diff
@@ -33,6 +33,7 @@
   },
   "dependencies": {
     "@enact/core": "^4.9.0-beta.1",
+    "@enact/i18n": "^4.9.0-beta.1",
     "classnames": "^2.5.1",
     "direction": "^1.0.4",
```
Comment (MikyungKim) [inline packages/ui/package.json:38]: Shound't we remove this?

File: packages/ui/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact ui module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `ui/Marquee.MarqueeDecorator` to use `@enact/i18n/utils.isRtlText` API.
```
Comment (MikyungKim) [inline packages/ui/CHANGELOG.md:9]:
```suggestion
- `ui/Marquee.MarqueeDecorator` to use `@enact/i18n/utils.isRtlText` API to determine the text directionality
```

Comment (seunghoh) [inline packages/ui/CHANGELOG.md:9]: Will this change affect app developers?  Should we add this to the changelog?

File: packages/ui/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact ui module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
```
[the remark this answers is not in the document]
Comment (hong6316) [reply packages/ui/CHANGELOG.md:7]: OK

File: packages/ui/CHANGELOG.md (hunk 4 above)
Comment (hong6316) [reply packages/ui/CHANGELOG.md:9]: OK

File: packages/ui/Marquee/MarqueeDecorator.js (hunk 1 above)
Comment (hong6316) [reply packages/ui/Marquee/MarqueeDecorator.js:3]: OK

File: packages/ui/Marquee/MarqueeDecorator.js (hunk 2 above)
Comment (hong6316) [reply packages/ui/Marquee/MarqueeDecorator.js:128]: OK

File: packages/ui/package.json (hunk 3 above)
Comment (hong6316) [reply packages/ui/package.json:38]: OK

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by seunghoh — APPROVED** [review]

**Pull request merged**